src/run.h

```diff
@@ -101,6 +109,8 @@ class Run {
     kj::ForkedPromise<void> startedFork;
     kj::PromiseFulfillerPair<RunState> finished;
     kj::ForkedPromise<RunState> finishedFork;
+
+    std::map<std::string, std::string> metaDataMap;
```

Comment:
There is a minor consideration here: what order should the tags show up in the frontend? Using `std::map` means they will show up alphabetically, but I wonder if it would be preferable for them to show up in the order that they were created? Unfortunately I think that would require using `vector<pair<string,string>>` and manually checking for duplicates, or using `boost::multi_index_container`

Reply:
Alphabetically is also fine